pkg/name/name.go

```diff
@@ -103,6 +103,22 @@ var (
 		KialiComponentName:              ThirdPartyFeatureName,
 		TracingComponentName:            ThirdPartyFeatureName,
 	}
+	// ComponentNameToHelmComponentPath defines mapping from component name to helm component root path.
```

Comment:
should be fairly easy to create this programmatically, maybe when you create a translator/reverse translator?

Reply:
The problem is that the checkenablement part does not depend on translator so I don’t want to get this mapping from the translator, not sure whether there is better way possibly?

Reply:
It probably should depend on it, or actually should go into Translator. Name is a very low level pkg and shouldn't know anything about things like helm so anyway this function probably doesn't belong there now.

Reply:
@ostromart translation is the process to convert between operator spec and helm values spec, however to get enablement values or some other paths from one of them does not reply on the other, right? I think it is ok to put them in names now as it does not need a process of translation. I add a TODO to refactor this part later but hope to get this merged soon to fix related bugs first.